src/energy_comms/extract/cps.py

```diff
@@ -0,0 +1,38 @@
+# imports
+
+import json
+
+import pandas as pd
+import prettytable
+import requests
+
+
+def extract_API():
+    headers = {"Content-type": "application/json"}
+    data = json.dumps(
+        {"seriesid": ["LNS14000000"], "startyear": "2009", "endyear": "2021"}
+    )
+    p = requests.post(
+        "https://api.bls.gov/publicAPI/v2/timeseries/data/", data=data, headers=headers
+    )
+    json_data = json.loads(p.text)
+    for series in json_data["Results"]["series"]:
```

Comment:
Instead of loading this into a pretty table, it would be simpler if we go straight to Pandas dataframe. I don't think we ever need the Pretty Table?

Picking up at line 19, this code produces the same table but without the Pretty Table:
```
df = pd.DataFrame()
for series in json_data["Results"]["series"]:
        series_df = pd.json_normalize(series["data"])
        series_df["series_id"] = series["seriesID"]
        df = pd.concat([df, series_df])
df = df[(df.period >= "M01") & (df.period <= "M12")]
return df
```